binding/rust/README.md

```diff
@@ -32,27 +30,26 @@ pv_recorder = "*"
 Getting the list of input devices does not require an instance:
 
 ```rust
-use pv_recorder::RecorderBuilder
+use pv_recorder::PvRecorderBuilder
 
-let audio_devices = RecorderBuilder::default().get_audio_devices()?;
+let audio_devices = PvRecorderBuilder::default().get_audio_devices()?;
 ```
 
 To start recording initialize an instance using the builder and run `start`:
 
 ```rust
-use pv_recorder::RecorderBuilder;
+use pv_recorder::PvRecorderBuilder;
 
-let recorder = RecorderBuilder::new().init()?;
+let recorder = PvRecorderBuilder::new(512).init()?;
```

Comment:
does rust have argument labels, i.e. `PvRecorderBuilder::new(frameLength: 512).init()?;` ?

If not, you should mention it in a comment or in the sentence above

Reply:
No argument labels, I'll set a variable